application/backend/app/api/routers/training_configurations.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from typing import Annotated
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, HTTPException, status
+
+from app.api.dependencies import get_training_configuration_service
+from app.configuration_tools.training_configuration_converter import convert_training_configuration_to_rest
+from app.services import ResourceNotFoundError
+from app.services.training_configuration_service import TrainingConfigurationService
+
+logger = logging.getLogger(__name__)
+router = APIRouter(prefix="/api/projects/{project_id}/training_configuration", tags=["Training Configuration"])
+
+
+@router.get("")
+def get_training_configuration(
+    training_configuration_service: Annotated[
+        TrainingConfigurationService, Depends(get_training_configuration_service)
+    ],
+    project_id: UUID,
```

Comment:
```suggestion
    project_id: Annotated[UUID, Depends(get_project_id)],
```
Use dependencies to make sure the parameter is properly validated.